mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/utbetaling/db/DelutbetalingQueries.kt

```diff
@@ -72,6 +73,52 @@ class DelutbetalingQueries(private val session: Session) {
         return session.requireSingle(queryOf(query, tilsagnId)) { it.int("lopenummer") }
     }
 
+    fun getSkalSendesTilOkonomi(tilsagnId: UUID): List<DelutbetalingDto> {
+        @Language("PostgreSQL")
+        val query = """
+            select
+                tilsagn_id,
+                utbetaling_id,
+                belop,
+                periode,
+                lopenummer,
+                fakturanummer,
+                opprettet_av,
+                created_at,
+                besluttet_av,
+                besluttet_tidspunkt,
+                besluttelse,
+                aarsaker,
+                forklaring
+            from delutbetaling
+            where
+                tilsagn_id = :tilsagn_id
+                and sendt_til_okonomi_tidspunkt is null
+                and besluttelse = 'GODKJENT'
+            order by besluttet_tidspunkt asc
+        """.trimIndent()
+
+        return session.list(queryOf(query, mapOf("tilsagn_id" to tilsagnId))) { it.toDelutbetalingDto() }
+    }
+
+    fun setSendtTilOkonomi(utbetalingId: UUID, tilsagnId: UUID) {
+        @Language("PostgreSQL")
+        val query = """
+            update delutbetaling set
+                sendt_til_okonomi_tidspunkt = now()
```

Comment:
Blir dette "riktig" tidspunkt? Vi har vel det caset nå med at `LocalDateTime`  fra applikasjonen og `timestamp` i db har litt tidssone-foryskyvning. Vi burde nok ta en grundig gjennomgang her etter hvert, men jeg tror det tryggeste er å sende tidspunktet fra applikasjonen i stedet for å bruke `now()` i sql-spørringen

Reply:
(Det kan potensielt gjøre testing enklere om man kan styre tidspunktet selv også)

Reply:
fikset